themes/digital.gov/layouts/communities/list.html

```diff
@@ -1,123 +1,23 @@
 {{ define "content" }}
-  <main role="main" id="main-content">
-    <div class="grid-container grid-container-desktop">
-      <div class="grid-row tablet-lg:grid-gap-4">
-        <div class="grid-col-12">
-          <header class="page-head page-head-{{- .Type -}}">
-            {{/* Page Title */}}
-            <h1>{{- .Title | markdownify -}}</h1>
-
-            {{/* Deck */}}
-            {{- if .Params.deck -}}
-              <div class="deck">{{- .Params.deck | markdownify -}}</div>
-            {{- end -}}
-
-            {{/* {{ partial "last-updated.html" . }} */}}
-          </header>
-        </div>
+  {{/* Gets all community pages and filter out those that are not the featured dg top 7 */}}
+  {{ $communities := (where (where .Site.Pages "Section" "communities").ByTitle ".Params.weight" "ge" 1) }}
+  {{ $other_communities := (where $communities ".Params.dg_highlight" "!=" true) }}
+
+
+  <main class="grid-container-desktop" id="main-content">
+    {{/* Digital.gov Communities of Practice */}}
+    {{ partial "core/featured-communities.html" (dict "context" . "layout" "has-image") }}
```

Comment:
For displaying images, left justified text and outline button styles, use the `has-image` variant.

```
{{ partial "core/featured-communities.html" (dict "context" . "layout" "has-image") }}
``` 

Default component does not need this setting to display no image, centered text and default button styles.